lib/nerves_hub/device_transport.ex

```diff
@@ -0,0 +1,69 @@
+defmodule NervesHub.DeviceTransport do
```

Comment:
```suggestion
defmodule NervesHub.DeviceSSLTransport do
```
Maybe we want SSL in there somewhere? Im good either way

Reply:
+1 on adding `SSL`